app/views/peeps/all.erb

```diff
@@ -0,0 +1,17 @@
+<ul>
+  <% @peeps.each do |peep| %>
+    <li class="peep" id="peep-<%= peep.id %>">
+      <%= peep.text%>
```

Comment:
You could make the formatting a little more consistent - tho it's not raising any errors and a good linter could correct automatically.  eg consistent spacing between erb interpolation tags <%= hi %>. Also consider closing the self-closing html tags for readability  <input ... />